android/build.gradle

```diff
@@ -18,7 +18,7 @@ android {
         minSdkVersion 16
         targetSdkVersion 31
         versionCode 1
-        versionName '3.5.6'
+        versionName '3.5.7'
```

Comment:
The latest release for Android is 3.5.6

Reply:
this version code is for react native sdk, not android sdk version. do you want to leave the version code 3.5.6?